draft-ietf-scitt-architecture.md

```diff
@@ -139,9 +137,9 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
 This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
 It achieves this goal by enforcing the following complementary security guarantees:
 
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
-1. Such Statements must be registered on a secure Append-only Log, enabling provenance and history to be independently and consistently audited
-1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
+1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable.
```

Comment:
Throughout the doc, we removed punctuation on single-sentence lines. If we want change that, we should do a single sweep for consistency

Reply:
> Throughout the doc, we removed punctuation on single-sentence lines.
I am relatively certain this is not true :sweat_smile: Two examples to counter: line 201 (was 203) and line 386 (was 383).
What I did here was just to add periods to complete sentences that appear in itemized lists.